src/percy-agent-client/dom.ts

```diff
@@ -169,7 +169,9 @@ class DOM {
       // rerendered and do not effect the visuals of a page
       if (clonedDOM.head.contains(cloned)) {
         cloned!.remove()
-
+      // Remove any lazy loading attributes, since they hang asset discovery
+      } else if (cloned!.getAttribute('loading')) {
```

Comment:
This will prevent the following conditionals from being encountered, thus not being able to serialize accessible frames that have this attribute. We should pop this out into it's own block before or after all of these other conditionals.

Reply:
🤦 Duh